Felienne/hedy
# Add quiz data to admin and class stats pages #1787

**Description**

The following metrics are added to the class stats and admin stats pages:
- number completed quizzes per level and per week
- number of abandoned quizzes per level and per week
- max, min and avg quiz score per level and per week

**Fix for**

#1787 

**How to test**

Admin page
Create test data via completing (and not completing) quizzes for different levels. The data created can be done for logged and anonymous users. Log in as an admin and go to /admin/stats. The admin page contains 2 new tabs: Quiz scores per level and Quiz scores per week. The latter has a shared legend.
- Check if the tabs of the accordion open and close as expected
- Check the data is displayed correctly
- Check that the shared legend works as expected

Class stats page
Create test data through completing and abandoning quizzes for different levels. Note that this time you need to do that via student accounts which are already in a class. Log in as a teacher and go to /stats/class/$class-id. New charts are added to the Students' overview tab.
- Check the data is displayed correctly in all quiz charts
- Check that the shared legend works of the quizzes as expected


**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
